reana_client/cli/utils.py

```diff
@@ -127,7 +127,34 @@ def parse_parameters(_filter):
         )
 
 
-def filter_data(parsed_filters, headers, tablib_data):
+def parse_filter_parameters(filters):
+    """Return parsed filter parameters."""
+    try:
+        status_filters = []
+        search_filters = None
+        filters = " ".join(filters).split(",")
+        for item in filters:
+            if "=" in item:
+                filter_name = item.split("=")[0].lower()
+                filter_value = item.split("=")[1]
+                if filter_name == "status":
+                    status_filters.append(filter_value)
+                elif filter_name == "name":
+                    search_filters = filter_value
+                else:
+                    click.echo("Filter {} is not supported yet.".format(filter_name))
```

Comment:
```suggestion
                    click.echo("Filter {} is not supported.".format(filter_name))
```

If someone types `--filter foo=bar`, let's not give them fake hope that we're gonna implement filter `foo` 😉

Reply:
Done 👍

Reply:
We can rather say that the filter is not valid.  We are doing it already for the `logs` command:

```console
$ reana-client logs -w helloworld-yadage --filter staaatus=foo
Error: filter 'staaatus' is not valid.
Available filters are 'compute_backend' 'docker_img' 'status' 'step'.
```

So we can reuse something identical here regarding validation. Aiming for the same user experience.

Reply:
Done 👍